LICENSE

```diff
@@ -1,4 +1,4 @@
-GNU GENERAL PUBLIC LICENSE
+                     GNU GENERAL PUBLIC LICENSE
```

Comment:
Hi @kshithijiyer, thank you for the contribution.
I would prefer to keep it `.md`, but I agree it requires formatting - we can just replace current [content with this one](https://github.com/TheFox/GPLv3.md/blob/master/gplv3.md). Please let me know if you'd like to reopen a PR, or I can do the changes myself